VoodooPS2Trackpad/VoodooPS2TrackpadCommon.h

```diff
@@ -11,6 +11,12 @@
 
 #define TEST_BIT(x, y) ((x >> y) & 0x1)
 
+#define DICT_SET_NUM(dict, key, num)                      \
+do {                                                      \
+  OSNumber *val = OSNumber::withNumber(num, 32);          \
+  if (val) { dict->setObject(key, val); val->release(); } \
+} while (0)
+
```

Comment:
Is this better as an inline function here? I'm not really sure what the best thing to do is for a helper function like this.

Reply:
Well, probably inline function is more C++ way, but I would say either is fine. Performance-wise the inline function should be the same too. If in doubt, I would go the inline function way.